api-specs/user.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package userapi;
+option go_package = "/userapi";
+
+service userService {
+	rpc getProfile (getProfileRequest) returns (Profile);
```

Comment:
thinking if we should separate get username/displayname and get friends as separate methods, possible usecase is user just made changes to their frens list and we need fetch list of frens again, wdyall think

Reply:
oops missed this, I'll modify as you suggested!